core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/util/JsonUtil.java

```diff
@@ -62,21 +64,21 @@ public static <T> T jsonToModel(JSONObject json, Class<T> modelClass) {
         return null;
     }
 
-    public static JSONArray getJsonArrayFromFile(String jsonPath, ResourceResolver resourceResolver) {
+    public static JsonArray getJsonArrayFromFile(String jsonPath, ResourceResolver resourceResolver) {
```

Comment:
To replace this whole method:

```suggestion
    public static ArrayNode getJsonArrayFromFile(String jsonPath, ResourceResolver resourceResolver) {
        Optional<InputStream> streamOptional = Optional.ofNullable(resourceResolver.getResource(jsonPath))
                .map(resource -> resource.adaptTo(InputStream.class));
        if (streamOptional.isPresent()) {
            try {
                String stringValue = IOUtils.toString(streamOptional.get(), StandardCharsets.UTF_8);
                return (ArrayNode) OBJECT_MAPPER.readTree(stringValue);
            } catch (IOException e) {
                LOG.error("Failed to get JSON array", e);
            }
        } else {
            LOG.debug("Failed to get json array from {}", jsonPath);
        }
        return OBJECT_MAPPER.createArrayNode();
    }
```

Reply:
done